CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Here is an overview of all **stable** additions:
 Here is an overview of all new **experimental** features:
 
 - **General**: TODO ([#TODO](https://github.com/kedacore/keda/issues/TODO))
+- **GitHub Scaler**: Introduced new GitHub Scaler (#1732)
```

Comment:
Is this in experimental section intentionally?

Reply:
I put it there as its very new. I'm just writing working e2e tests and happy to move from experimental if you prefer. Also making some changes based on the comments in keda-docs. When is 2.10 release?

Reply:
Tomorrow :)

We can keep this in experimental, it is not a problem wdyt @kedacore/keda-contributors ?

Reply:
I don't have any preference tbh, What do you propose? Releasing the feature without e2e tests as experimental?

Reply:
E2E are a must even for an experimental.

Reply:
That's why I asked because for me, they are, but... Having the e2e test, adding it as experimental or not is not a problem IMO. New features could fail

Reply:
I'm adding them right now